pipelines/concourse-pipelines.yml

```diff
@@ -0,0 +1,70 @@
+resource_types:
+
+  - name: concourse-pipeline
+    type: docker-image
+    source:
+      repository: concourse/concourse-pipeline-resource
+
+resources:
+
+  - name: census-rm-deploy
+    type: git
+    source:
+      uri: git@github.com:ONSdigital/census-rm-deploy.git
+      paths:
+      - "pipelines"
+      private_key: ((github.service_account_private_key))
+
+  - name: pipelines
+    type: concourse-pipeline
+    source:
+      teams:
+      - name: rm
+        username: rm
+        password: ((concourse.team-pass))
+
+jobs:
+
+  - name: fly-pipelines
+    plan:
+    - get: census-rm-deploy
+      trigger: true
+    - put: pipelines
+      params:
+        pipelines:
+
+        - name: concourse-pipelines
+          team: rm
+          config_file: census-rm-deploy/pipelines/concourse-pipelines.yml
+
+        - name: CI-Whitelodge Deploy
+          team: rm
+          config_file: census-rm-deploy/pipelines/ci-sit-kubernetes-pipeline.yml
+          vars:
+            ci-gcp-environment-name: ci
+            ci-gcp-project-name: census-rm-ci
+            ci-kubernetes-cluster-name: rm-k8s-cluster
+            terraform-branch: master
+            acceptance-tests-image: eu.gcr.io/census-gcr-rm/rm/census-rm-acceptance-tests:latest
+            sit-gcp-environment-name: whitelodge
+            sit-gcp-project-name: census-rm-whitelodge
+            sit-kubernetes-cluster-name: rm-k8s-cluster
+            ci-gcp-admin-account-json:
+            ci-gcp-service-account-json:
+            sit-gcp-service-account-json:
```

Comment:
These last three shouldn't be in use anymore?

Reply:
@AdamHawtin Do we now use a single service account for all projects including admin tasks?